### PR TITLE
Css brochure form - Minji Cho

### PR DESCRIPTION
**Pull request by m-Cho1**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Minji! This form is too wide as compared to the example, and the color of your header and button does not match the example either. Please revise, and submit a new screenshot with your revision.

**Comment by m-Cho1** [comment]
image

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <h4>Learn More</h4>
+    <form>
+      <div class="company-email">
+        <p class="company-email">company email</p>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:17]: This element should be the `label`. Right now you have an empty `label` surrounding your `input`, which won't do anything.

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <label class="email">
+          <input type="text" name="company-email" placeholder="sally@company.co" />
+        </label>
+      </div>
+      <div>
+        <p class="name">name</p>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:23]: This element should be the `label`. Right now you have an empty `label` surrounding your `input`, which won't do anything.

File: css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <label class="name">
+          <input type="text" name="name" placeholder="Sally Smith" />
+        </label>
+      </div>
+      <div class="place-container">
+        <p class="place">where are you hiring?</p>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:29]: This element should be the `label`. Right now you have an empty `label` surrounding your `input`, which won't do anything.

File: css-brochure-form/index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        <label class="place">
+          <input type="text" name="place" placeholder="City, Country" />
+        </label>
+      </div>
+      <div class="select-container">
+        <p class="hire-time">how many engineers are you hiring in the next 6 months?</p>
```
Comment (codingcodymiller) [inline css-brochure-form/index.html:35]: This element should be the `label`. Right now you have an empty `label` surrounding your `input`, which won't do anything.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by m-Cho1** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**